docs/source/examples/docker_recipes.md

```diff
@@ -0,0 +1,53 @@
+# Docker Recipes
+
+This page contains some recurrent implementation patterns used in Dockerfile when constructing workflows. 
```

Comment:
This page contains some common implementation patterns used in `Dockerfile`s when building images for workflows.